addons/calendar/models/res_users.py

```diff
@@ -81,11 +81,12 @@ def _compute_calendar_default_privacy(self):
         When any user doesn't have its setting from ResUsersSettings defined, fallback to Default User Template's.
         """
         fallback_default_privacy = 'public'
-        if any(not user.res_users_settings_id.calendar_default_privacy for user in self):
+        # sudo: user has access to others settings to avoid access issues in other modules.
```

Comment:
Proper explanation should be more like the others... Any user can see the default calendar privacy of other users, something like that.